test/test_nn.py

```diff
@@ -11351,6 +11390,116 @@ def test_maxpool3d_non_square_backward(self, device):
             self.assertTrue(torch.allclose(x.grad, torch.ones_like(x.grad)))
 
 
+class TestConv2dExt(TestCase):
+    def test_conv2d_ext(self):
+        def subtensor(tensor, dim, groups, g):
+            if tensor is None:
+                return None
+            group_size = int(tensor.size(dim) / group)
```

Comment:
Where does the `group` variable come from?

Also, if this should perform integer division then use the floor division (//) operator, not the division operator.